pkgs/_test_macros/lib/declare_x_macro.dart

```diff
@@ -3,6 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:dart_model/dart_model.dart';
+import 'package:dart_model/templating.dart';
```

Comment:
I would move this all the way out to either a testing utility or the JSON macro example itself etc.

I don't think we should be focusing on shipping a templating syntax as part of this, it isn't a requirement at all for shipping things. And long term probably shouldn't exist at all (we really want custom interpolations via tagged strings or similar).